locustempus/main/models.py

```diff
@@ -69,6 +69,7 @@ class MediaObject(models.Model):
     url = models.URLField()
     source = models.TextField(blank=True)
     caption = models.TextField(blank=True)
+    alt = models.TextField(blank=True)
```

Comment:
Interesting, in the past I have always set an optional TextField like this with both `blank=True` and `null=True`. It looks like this is actually recommended against now, and using only `blank=True` is best in most cases.

https://docs.djangoproject.com/en/4.1/ref/models/fields/#null